index.js

```diff
@@ -89,14 +100,34 @@ function lint(textEditor) {
 	});
 }
 
+function fix(editor) {
+	const filePath = editor.getPath();
+	const {pkg} = findPkg(pkgDir.sync(path.dirname(filePath)));
+
+	if (!editor || !xoInPkg(pkg)) {
```

Comment:
This is incorrect and inefficient. When the `editor` does not exist it will fail on line 104, as it tries to get `.getPath()` of `undefined`. There's also no point in running `findPkg` when `editor` does not exist. The editor if-statement should be at the top.